config_test.go

```diff
@@ -147,6 +147,69 @@ func TestConfigs_appPaths(t *testing.T) {
 	}
 }
 
+func TestConfigs_mappingPaths(t *testing.T) {
+	tests := []struct {
+		name        string
+		configs     Configs
+		wantErr     bool
+		createFiles []string
+	}{
+		{
+			name:    "no mapping file",
+			configs: Configs{},
+			wantErr: false,
+		},
+		{
+			name:        "single mapping file",
+			configs:     Configs{MappingFile: os.TempDir() + "TestConfigs_mappingPaths_mapping.txt"},
+			wantErr:     false,
+			createFiles: []string{os.TempDir() + "TestConfigs_mappingPaths_mapping.txt"},
```

Comment:
NOTE: Please use filepath.Join to join path, that should work in every scenario and different platforms even.